.github/workflows/cd.yml

```
@@ -0,0 +1,171 @@
+
+jobs:
+  # Build and push Docker image
+  build-and-push:
+    name: Build and Push Docker Image
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    outputs:
+      image-digest: ${{ steps.build.outputs.digest }}
+      image-url: ${{ steps.build.outputs.image-url }}
```

actionlint (1.7.7)

property "image-url" is not defined in object type {digest: string; imageid: string; metadata: string}

(expression)